### PR TITLE
HDXDSYS-1387 Add to all HAPI views a generated column: admin_level 

### PR DESCRIPTION
**Pull request by mcarans**

### CODE REVIEW

File: src/hapi_schema/db_conflict_event.py [hunk 1]
```diff
@@ -77,6 +80,29 @@ class DBConflictEvent(Base):
         DBAdmin2.name.label("admin2_name"),
         DBAdmin2.is_unspecified.label("admin2_is_unspecified"),
         DBAdmin2.admin1_ref.label("admin1_ref"),
+        case(
```
Comment (IanHopkinson) [inline src/hapi_schema/db_conflict_event.py:83]: This code appears repeatedly, I've wondering if there is a way of factoring it out

**Review by IanHopkinson — APPROVED** [review]
I think like tests should have been updated for this change, in particular to provide confirmation that the new column has been added to all relevant views.

It would be useful to include the ticket/epic under which this work is being done in order to understand how it fits in with work required on the HAPI repo.

File: src/hapi_schema/db_conflict_event.py (hunk 1 above)
Comment (mcarans) [reply src/hapi_schema/db_conflict_event.py]: I have done so

**Comment by mcarans** [comment]
@IanHopkinson Where did you have in mind to add the ticket (which is already referenced in the PR title)?

**Comment by IanHopkinson** [comment]
> @IanHopkinson Where did you have in mind to add the ticket (which is already referenced in the PR title)?

Ooops! I didn't spot that - I reviewed early in my morning. I guess my habit is usually to provide more description for a PR

**Comment by IanHopkinson** [comment]
@mcarans the factoring and testing look good to me!

**Pull request merged**